src/index.js

```diff
@@ -40,7 +40,7 @@ const stripOneCapture = ({blob, documentType, onfidoId, side}) => {
 const stripCapturesHash = captures => mapValues(captures,
   capture => capture ? stripOneCapture(capture) : null)
 
-const getCaptures = () => stripCapturesHash(events.getCaptures())
+const getCaptures = () => mapKeys(stripCapturesHash(events.getCaptures()), key => key + 'Capture')
```

Comment:
public api together with all the necessary compatibility/formatting of output/input data is done mostly here, feels weird to have it somewhere else.